v3/src/components/graph/models/graph-data-configuration-model.ts

```diff
@@ -397,26 +397,18 @@ export const GraphDataConfigurationModel = DataConfigurationModel
         }
       })
     },
-    binDetails(_binAlignment?: number, _binWidth?: number, reset = false) {
+    binDetails(_binAlignment?: number, _binWidth?: number) {
       const kDefaultNumberOfBins = 4
-
       const minValue = self.caseDataArray.reduce((min, aCaseData) => {
         return Math.min(min, self.dataset?.getNumeric(aCaseData.caseID, self.primaryAttributeID) ?? min)
       }, Infinity)
-
       const maxValue = self.caseDataArray.reduce((max, aCaseData) => {
         return Math.max(max, self.dataset?.getNumeric(aCaseData.caseID, self.primaryAttributeID) ?? max)
       }, -Infinity)
 
       if (minValue === Infinity || maxValue === -Infinity) {
-        return { binAlignment: 0,
-          binWidth: 1,
-          minBinEdge: 0,
-          maxBinEdge: 0,
-          minValue: 0,
-          maxValue: 0,
-          totalNumberOfBins: 0 
-        }
+        return { binAlignment: 0, binWidth: 1, minBinEdge: 0, maxBinEdge: 0, minValue: 0, maxValue: 0,
+                 totalNumberOfBins: 0 }
       }
 
```

Comment:
When minValue === maxValue bad things happen, including that binWidth gets a value that the JSON parser chokes on during a restore. It's not clear what the best way to handle it is though.